src/applications/disability-benefits/disability-rating-calculator/components/DisabilityRatingCalculator.jsx

```diff
@@ -143,7 +143,13 @@ export default class DisabilityRatingCalculator extends React.Component {
           <p>
             Enter each of your disability ratings separately below. You can also
             add a description of each for your notes, if you'd like. Then click{' '}
-            <strong>Calculate</strong> to get your combined rating.
+            <strong>Calculate</strong> to get your combined rating.{' '}
+            <strong className="vads-u-display--inline small-screen:vads-u-display--none">
+              Disability ratings are given in 10% increments, from 0 to 100.
+            </strong>
+            <span className="vads-u-display--none small-screen:vads-u-display--inline">
+              Disability ratings are given in 10% increments, from 0 to 100.
+            </span>
```

Comment:
How come we want both <strong> and <span> for desktop/mobile differences? 🙂

Reply:
Oh I see, looks like it is this way in the design spec: 
![image](https://user-images.githubusercontent.com/12773166/129916457-cfd3b217-8e82-4074-b955-37f4771232cf.png)